src/app/legacy/containers/SocialEmbed/index.test.jsx

```diff
@@ -122,6 +124,44 @@ describe('SocialEmbedContainer', () => {
       unmount();
     });
 
+    it('should render a Facebook Post block and unmount correctly', () => {
+      const { container, unmount } = render(
+        <SocialEmbedContainer
+          blocks={[facebookPostBlockEmbed]}
+          source="https://www.facebook.com/RickAstley/posts/545713756920775"
+        />,
+        { service: 'news', isAmp: false, pageType: ARTICLE_PAGE },
+      );
+
+      expect(container.firstChild).toMatchSnapshot();
+
+      expect(loggerMock.info).toHaveBeenCalledTimes(1);
+      expect(loggerMock.info).toHaveBeenCalledWith(SOCIAL_EMBED_RENDERED, {
+        provider: 'facebook',
+        href: 'https://www.facebook.com/RickAstley/posts/545713756920775',
+      });
+      unmount();
```

Comment:
Are these unmounts meant to be used to explicitly test the component has unmounted correctly? If so we may need to check the [`container.innerHTML === ""`](https://testing-library.com/docs/react-testing-library/api/#unmount), I can see all the other tests don't have this just wanted to check

Reply:
Good spot actually, its missing a check that the script tags are removed after `unmount()` is called:
```
      expect(
        document.querySelector(
          'head script[src="https://www.instagram.com/embed.js"]',
        ),
      ).toBeFalsy();
```

I'll add that in.